.helm_production.yml

```diff
@@ -0,0 +1,34 @@
+ingress:
+  globalStaticIpName: bblfsh-dashboard-production 
+  hostname: play.bblf.sh
+
+nodeSelector:
+  cloud.google.com/gke-nodepool: default-pool
+
+bblfshd-sidecar:
+  drivers:
+    languages:
+      python:
+        repository: bblfsh/python-driver
+        tag: v2.4.0
+      java:
+        repository: bblfsh/java-driver
+        tag: v2.3.0
+      javascript:
+        repository: bblfsh/javascript-driver
+        tag: v2.3.0
+      typescript:
```

Comment:
could you please remove typescript? We agreed to use only beta drivers. And this one is [alpha](https://doc.bblf.sh/languages.html) which means it's kind of broken.